bedrock/mozorg/templates/mozorg/about/governance/policies/commit/access-policy.html

```diff
@@ -100,10 +100,12 @@ <h2 id="Level3">Level 3 - Core Product Access</h2>
 
 <p>Someone can be upgraded from level 2 to level 3 by being vouched for by a single module owner of level-3-stored code.</p>
 
-<p>This permission gives access to check into any tree from which executable code becomes part of our core products - Firefox, Firefox for Android and Thunderbird. To put it another way, the unifying factor is that it should not be possible to break core product tinderboxes unless you have this access. This is fundamentally a statement of trust in and familiarity with an individual, and so access to one such tree gives access to all such trees, although social controls may prevent people checking in to certain ones. (Peers can vouch because the number of modules at this level is smaller, and because if you are working only in a single area of the code, there may not be multiple module owners familiar with your work.)</p>
+<p>This permission gives access to check into any tree from which executable code becomes part of our core products - Firefox, GeckoView and Thunderbird. This is fundamentally a statement of trust in and familiarity with an individual, and so access to one such tree gives access to all such trees, although social controls may prevent people checking in to certain ones.</p>
 
 <p>People with this access may not actually be working on Firefox, Firefox for Android or Thunderbird, but their ability to affect them means that the level of trust and familarity required for access are higher.</p>
```

Comment:
As you changed `Firefox for Android` to `GeckoView` in the previous point, the same replacement should happen here.